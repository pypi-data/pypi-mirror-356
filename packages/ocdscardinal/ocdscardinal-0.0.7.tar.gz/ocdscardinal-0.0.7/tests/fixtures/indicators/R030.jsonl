{"ocid":"F","bids":{"details":[{"status":"valid","date":"2000-04-01T00:00:00Z","tenderers":[{"id":"W"}]}]},"tender":{"tenderPeriod":{"endDate":"2000-03-15T00:00:00Z"}},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"losing bid","bids":{"details":[{"status":"valid","date":"2000-04-01T00:00:00Z","tenderers":[{"id":"L"}]}]},"tender":{"tenderPeriod":{"endDate":"2000-03-15T00:00:00Z"}},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
{"ocid":"pending award","bids":{"details":[{"status":"valid","date":"2000-04-01T00:00:00Z","tenderers":[{"id":"W"}]}]},"tender":{"tenderPeriod":{"endDate":"2000-03-15T00:00:00Z"}},"awards":[{"status":"pending","suppliers":[{"id":"W"}]}]}
{"ocid":"invalid award","bids":{"details":[{"status":"valid","date":"2000-04-01T00:00:00Z","tenderers":[{"id":"W"}]}]},"tender":{"tenderPeriod":{"endDate":"2000-03-15T00:00:00Z"}},"awards":[{"status":"invalid","suppliers":[{"id":"W"}]}]}
{"ocid":"cancelled tender","bids":{"details":[{"status":"valid","date":"2000-04-01T00:00:00Z","tenderers":[{"id":"W"}]}]},"tender":{"tenderPeriod":{"endDate":"2000-03-15T00:00:00Z"},"status":"cancelled"},"awards":[{"status":"active","suppliers":[{"id":"W"}]}]}
