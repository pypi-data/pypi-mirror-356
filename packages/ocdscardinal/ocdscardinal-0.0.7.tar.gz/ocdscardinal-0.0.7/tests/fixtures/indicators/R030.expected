{"OCID":{"F":{"R030":1.0}},"Tenderer":{"W":{"R030":1.0}}}
